network-mux/src/Network/Mux/Types.hs

```diff
@@ -291,14 +291,14 @@ instance Exception MuxError where
 
 -- Type used for tracing mux events.
 --
-data WithMuxBearer a = WithMuxBearer {
-      wmbTag   :: !String
+data WithMuxBearer peerid a = WithMuxBearer {
+      wmbTag   :: !peerid
```

Comment:
I would rename it to `wmbPeerId` as well, hopefuly there are not too many places where it is used.